### PR TITLE
[New Rule] 2 Rules for Persistence via Emond

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/831

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Review by peasead — APPROVED** [review]

File: rules/macos/persistence_emond_rules_file_creation.toml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+references = ["https://www.xorrior.com/emond-persistence/"]
+risk_score = 47
+rule_id = "a6bf4dd4-743e-4da8-8c03-3ebd753a6c90"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
+file where event.type != "deletion" and
+ wildcard(file.path, "/private/etc/emond.d/rules/*.plist", "/etc/emon.d/rules/*.plist")
```
Comment (bm11100) [inline rules/macos/persistence_emond_rules_file_creation.toml:26]:
```suggestion
file where event.type != "deletion" and
 file.path:("/private/etc/emond.d/rules/*.plist", "/etc/emon.d/rules/*.plist")
```

Comment (bm11100) [reply rules/macos/persistence_emond_rules_file_creation.toml:26]: eliminate wildcard function

File: rules/macos/persistence_emond_rules_process_execution.toml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+risk_score = 47
+rule_id = "3e3d15c6-1509-479a-b125-21718372157e"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Execution", "Persistence"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+ process.parent.name == "emond"  and
+ process.name in ("bash", "dash", "sh", "tcsh", "csh", "zsh", "ksh", "fish", "Python", "python*", "perl*", "php*", "osascript", "pwsh", "curl", "wget", "cp", "mv", "touch", "echo", "base64", "launchctl")
```
Comment (bm11100) [inline rules/macos/persistence_emond_rules_process_execution.toml:28]:
```suggestion
process where event.type in ("start", "process_started") and
 process.parent.name:"emond" and
 process.name:(
   "bash",
   "dash",
   "sh",
   "tcsh",
   "csh",
   "zsh",
   "ksh",
   "fish",
   "Python",
   "python*",
   "perl*",
   "php*",
   "osascript",
   "pwsh",
   "curl",
   "wget",
   "cp",
   "mv",
   "touch",
   "echo",
   "base64",
   "launchctl")
```

Comment (bm11100) [reply rules/macos/persistence_emond_rules_process_execution.toml:28]: readability and use of `:`
```
process where event.type in ("start", "process_started") and
 process.parent.name:"emond" and
 process.name:(
   "bash",
   "dash",
   "sh",
   "tcsh",
   "csh",
   "zsh",
   "ksh",
   "fish",
   "Python",
   "python*",
   "perl*",
   "php*",
   "osascript",
   "pwsh",
   "curl",
   "wget",
   "cp",
   "mv",
   "touch",
   "echo",
   "base64",
   "launchctl")
```

**Review by bm11100 — COMMENTED** [review]
couple suggestions, otherwise looks good

File: rules/macos/persistence_emond_rules_file_creation.toml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+references = ["https://www.xorrior.com/emond-persistence/"]
+risk_score = 47
+rule_id = "a6bf4dd4-743e-4da8-8c03-3ebd753a6c90"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
+file where event.type != "deletion" and
+ file.path:("/private/etc/emond.d/rules/*.plist", "/etc/emon.d/rules/*.plist")
```
Comment (bm11100) [inline rules/macos/persistence_emond_rules_file_creation.toml:26]:
```suggestion
 file.path : ("/private/etc/emond.d/rules/*.plist", "/etc/emon.d/rules/*.plist")
```

File: rules/macos/persistence_emond_rules_process_execution.toml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+risk_score = 47
+rule_id = "3e3d15c6-1509-479a-b125-21718372157e"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Execution", "Persistence"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+ process.parent.name:"emond" and
+ process.name:(
```
Comment (bm11100) [inline rules/macos/persistence_emond_rules_process_execution.toml:28]:
```suggestion
 process.parent.name : "emond" and
 process.name : (
```

**Review by bm11100 — APPROVED** [review]

**Pull request merged**